### PR TITLE
Update pkg/apis with new generated controllers

### PR DESCRIPTION
**Pull request by JonCrowther**
## Issue: https://github.com/rancher/rancher/issues/47972
Assists https://github.com/rancher/rancher/pull/47997 

## Problem
The PR to remove clusterlogging and projectlogging is failing CI because Shepherd has references to those CRDs
 
## Solution

A little tricky, but this is the rough workflow:

1. Point shepherd to use pkg/apis from my rancher/rancher branch
2. Update rancher/rancher PR to use this branch
3. Merge rancher/rancher PR
4. Update shepherd PR to use official rancher/rancher branch
5. Merge shepherd PR
6. Create a new rancher/rancher branch to point to official shepherd branch
 
## Testing
No tests use clusterlogging or projectlogging resources, so no tests should be affected.

### Regressions Considerations

_TODO_

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -6,6 +6,7 @@ toolchain go1.22.7
 
 replace (
 	github.com/docker/distribution => github.com/docker/distribution v2.8.2+incompatible // rancher-machine requires a replace is set
+	github.com/rancher/rancher/pkg/apis => github.com/JonCrowther/rancher/pkg/apis v0.0.0-20241111203128-b46183871062 // using remove-cluster-project-logging branch
```
Comment (JonCrowther) [inline go.mod:9]: Not to be merged until this is resolved

**Comment by caliskanugur** [comment]
Could we get a squash for this?

**Comment by pmatseykanets** [comment]
> Could we get a squash for this?
It can be squashed on merge, which is what Jonathan most likely intents to do.

**Comment by JonCrowther** [comment]
I will squash and merge once it's ready

**Pull request merged**